metricbeat/module/zookeeper/_meta/supported-versions.yml

```diff
@@ -0,0 +1,7 @@
+variants:
+  - ZOOKEEPER_VARIANT: zookeeper
```

Comment:
`ZOOKEEPER_VARIANT` not needed?

Reply:
Actually not, I thought that field was mandatory but I've tested removing it and it still works 2060c082ac712c3d579a7a9234eb0acd81583a1d